### PR TITLE
Add usage documentation for generate profile command

### PR DESCRIPTION
**Pull request by martina-if**
Add documentation for `eksctl generate profile`

- [x] Added/modified documentation as required (such as the `README.md`, and `examples` directory)
- [x] Complete example with real output when the base repo is complete

### CODE REVIEW

File: site/content/usage/experimental/01-gitops.md [hunk 1]
```diff
@@ -92,14 +92,54 @@ memcached-958f745c-qdfgz   1/1     Running   0          29m
...
 
 To learn more about GitOps and Flux, check the [Flux documentation][flux]
  
 
+### Installing components from a quickstart
+
+`eksctl` provides an application development quickstart profile which can install the following components in your
+cluster: 
+  - Metrics Server
+  - Cortex/dynamo
```
Comment (cPu1) [inline site/content/usage/experimental/01-gitops.md]: We haven't added Cortex with DynamoDB support yet. We should replace this with Prometheus.

File: site/content/usage/experimental/01-gitops.md [hunk 2]
```diff
@@ -92,14 +92,54 @@ memcached-958f745c-qdfgz   1/1     Running   0          29m
...
+  - Grafana
+  - Kubernetes Dashboard
+  - FluentD with connection to cloudwatch logs
```
Comment (marccarre) [inline site/content/usage/experimental/01-gitops.md]: Nit: `cloudwatch` -> `CloudWatch`

File: site/content/usage/experimental/01-gitops.md [hunk 3]
```diff
@@ -92,14 +92,54 @@ memcached-958f745c-qdfgz   1/1     Running   0          29m
...
 To learn more about GitOps and Flux, check the [Flux documentation][flux]
  
 
+### Installing components from a quickstart
+
+`eksctl` provides an application development quickstart profile which can install the following components in your
+cluster: 
+  - Metrics Server
+  - Cortex/dynamo
+  - Grafana
```
Comment (marccarre) [inline site/content/usage/experimental/01-gitops.md]: We don't have this yet, afaik.

Comment (marccarre) [reply site/content/usage/experimental/01-gitops.md]: Ah, had missed it.

File: site/content/usage/experimental/01-gitops.md [hunk 4]
```diff
@@ -92,14 +92,54 @@ memcached-958f745c-qdfgz   1/1     Running   0          29m
...
+For example:
+
+<!-- TODO -->
+
+After running the command, add, commit and push the files:
+
+```bash
+cd my-gitops-repo/
+git add .
+git commit -m"Add application development quickstart components"
```
Comment (marccarre) [inline site/content/usage/experimental/01-gitops.md]: Nit: `git commit -m"` -> `git commit -m "` for better readability?

Comment (marccarre) [inline site/content/usage/experimental/01-gitops.md]: Nit: `git commit -m"` -> `git commit -m "` for better readability?

**Pull request merged**